src/synthesizer/abundances.py

```diff
@@ -16,7 +16,88 @@
 import matplotlib.pyplot as plt
 import matplotlib.cm as cm
 
-from .exceptions import InconsistentParameter
+from synthesizer.exceptions import (
+    InconsistentParameter,
+    UnrecognisedOption
+    )
+
+
+class ScalingFunctions:
+
+    """
+    This is a class holds scaling functions for individual elements. In each
+    case the function returns the logarthimic abundance relative to Hydrogen
+    for a given metallicity.
+    
+    Example:
+
+    ScalingFunctions.N.Dopita2006(0.016)
```

Comment:
A given publication / model may include many scaling relations, so wouldn't it be better to have `ScalingFunction.Dopita2006.N`  ?

Reply:
Possibly, though I think it's less intuitive to call then.

Reply:
However, that would allow us to add an additional attribute, like the paper DOI. So, yes, I think you're probably right.

Reply:
I've done this now.